client/src/components/PreviewImageField/PreviewImageField.js

```diff
@@ -302,7 +307,13 @@ class PreviewImageField extends Component {
           {message.value}
         </div>
       );
-    } else if (progress === 100) {
+    } else if (errors) {
+      messageBox = (
+        <div className={`preview-image-field__message preview-image-field__message--${errors.type}`}>
+          {errors.value}
```

Comment:
```suggestion
    } else if (errors) {
    const errorMessage = errors.value ? errors.value : i18n._t('AssetAdmin.DROPZONE_RESPONSE_ERROR', 'Server responded with an error.');
      messageBox = (
        <div className={`preview-image-field__message preview-image-field__message--${errors.type}`}>
          {errorMessage}
```
Sometimes there is no error message in the response, for example if you upload a file that is within the size limits in yml config, but too large for php upload limits.